crates/trident/src/engine/rollback.rs

```diff
@@ -117,6 +118,17 @@ pub fn validate_boot(datastore: &mut DataStore) -> Result<BootValidationResult,
                 servicing_type,
             );
         }
+        (true, ServicingState::ManualRollbackFinalized) => {
```

Comment:
this should be merged in with abupdate and clean install so encryption gets updated